docs/Model_Assumptions.rst

```diff
@@ -176,12 +176,117 @@ The confersion factors are derived from their `Gasoline Gallon Equivalents.<http
+:Implications: The perfect foresight can lead to suspicious dispatch of assets, for example charging of a battery right before a (in real-life) random blackout occurs. The systems optimized with the MVS therefore, represent their optimal potential, which in reality could not be reached. The MVS has thus a tendency to underestimate the needed battery capacity or the minimal state of charge for backup purposes, and also designs the PV system and backup power according to perfect forecasts. In reality, operational margins would need to be added.
 
-- Energy consumption and feed-in at the same time
-- Bi-directional inverters
+Extension of KPIs
+#################
+:Limitation: Some important KPIs, usually required by developers, are currently not implemented in the MVS.
+:Reason: The MVS tool is a work in progress and this can still be addressed in the future.
+:Implications: The absence of such indicators might affect decision-making.
+
+Random excess energy distribution
+#################################
+:Limitation: There is random excess distribution between the feed-in sink and the excess sink when no feed-in-tariff is assumed in the system.
+:Reason: Since there is no feed-in-tariff to benefit from, the MVS randomly distributes the excess energy between the feed-in and excess sinks.
+:Implications: This distribution of excess energy changes when running several simulations for the same input files.
+
+Renewable energy share defintion
+################################
+:Limitation: The current renewable energy share depends on the share of renewable energy production assets directly feeding the load.
+:Reason: The MVS tool is a work in progress and this can still be addressed in the future.
+:Implications: This might result in different values when comparing them to other models.
+
+Energy carrier rating
+#####################
+:Limitation: The MVS assumes a usable potential/energy content rating for every energy carrier.
+:Reason: This is an approach that the MVS currently uses.
+:Implications: By weighing the energy carriers according to their energy content (Gasoline Gallon Equivalent (GGE)), the MVS might result in values that can't be directly assessed. Those ratings affect the calaculation of the levelized cost of the energy carriers, but also the minimum renewable energy share constraint.
+
+Energy shortage or grid interruption
+####################################
+:Limitation: The MVS assumes no shortage or grid interruption in the system.
+:Reason: 
+:Implications: Electricity shortages due to power cuts might happen in real life. The MVS currently omits this scenario which might implicate unreasonable results. If a system is self-sufficient but relies on grid-connected PV systems, the latter stop feeding the load if any power cuts occur and the battery storage systems might not be enough to serve the load (energy shortage).
```

Comment:
As we assume that the grid is always available, this is true... but not something that we have to worry about. You can keep this in the limitations (maybe I will move it to the assumptions). However, I disagree that it `implicates unreasonable results` - this degrades the MVS, eventhough the MVS does not even aim to cover those scenarios...

Reply:
alright, I removed the sentence and added a reason. maybe it's better to remove it from the limitations then..